tests/phpunit/api/v3/SyntaxConformanceTest.php

```diff
@@ -573,6 +574,7 @@ public static function toBeSkipped_getSqlOperators() {
       'Extension', // can't handle creating 25
       'Note', // note has a default get that isn't implemented in createTestObject -meaning you don't 'get' them
       'Setting', //a bit of a pseudoapi - keys by domain
+      'Exception', // DB issue.
```

Comment:
erm - what DB issue

Reply:
It seems that it struggles to fill in all the fields sensibly. iirc it was easier to skip